src/offline/Makefile

```diff
@@ -165,7 +168,25 @@ CINC = -I$(NCMOD)
 	$(FC) $(CFLAGS) $(CINC) -c $<
 
 # default target by convention is ``all''
-all : 
+all: serial
+
+serial: supr = -warn nostderrors -diag-disable 10145
+serial: FOBJS = $(shell ls *.o)
+serial: cable_driver.F90 common
+	rm -f cable_driver.o
+	$(FC) $(supr) $(CFLAGS) $(LDFLAGS) $(LD) -o $(PROG_SERIAL) cable_driver.F90 $(FOBJS) $(CINC)
+
+mpi: supr = -warn nostderrors -diag-disable 10145
+mpi: FOBJS = $(shell ls *.o)
+mpi: cable_mpidrv.F90 cable_mpicommon.F90 pop_mpi.F90 cable_mpiworker.F90 cable_mpimaster.F90 common
+	rm -f cable-mpi
+	rm -f cable_mpi*.o
+	rm -f pop_mpi.o
+	$(FC) $(supr) $(CFLAGS) $(LDFLAGS) -c cable_mpicommon.F90 $(FOBJS) $(CINC)
+	$(FC) $(supr) $(CFLAGS) $(LDFLAGS) -c pop_mpi.F90 cable_mpicommon.o $(FOBJS) $(CINC)
+	$(FC) $(supr) $(CFLAGS) $(LDFLAGS) -c cable_mpiworker.F90 cable_mpicommon.o pop_mpi.o $(FOBJS) $(CINC)
+	$(FC) $(supr) $(CFLAGS) $(LDFLAGS) -c cable_mpimaster.F90 cable_mpicommon.o cable_mpiworker.o pop_mpi.o $(FOBJS) $(CINC)
+	$(FC) $(supr) $(CFLAGS) $(LDFLAGS) -o $(PROG_MPI) cable_mpidrv.F90 cable_mpicommon.o cable_mpimaster.o cable_mpiworker.o pop_mpi.o $(FOBJS) $(CINC) $(LD)
```

Comment:
We should leverage Makefile syntax here so that:
1. We do not have to remove object files or executable files.
2. We have a single build command for the serial and mpi targets that contains the correct list of object files.